packages/react-keytips/README.md

```diff
@@ -2,10 +2,29 @@
 
 ## Installation
 
+```bash
+npm install @fluentui-contrib/react-keytips
+```
+
+or
+
 ```bash
 yarn add @fluentui-contrib/react-keytips
 ```
 
+A Keytip is a small popup displayed near a UI component, indicating a key sequence that can be used to navigate to or trigger that component. Unlike keyboard shortcuts,
+Keytips guide users through a sequence of keys to traverse hierarchical levels of the UI.
+
+From a technical perspective, a Keytip is a wrapper around the Tooltip component.
+
+To enable Keytips, include the Keytips component at the root level of your application. Use the `useKeytipRef`
+hook to attach a Keytip to a specific target element.
+
+Users can toggle Keytip mode using the `startSequence` (Option+Command on macOS, Alt+Windows (Meta) on Windows, by default).
+To navigate back to the previous level of Keytips, the `returnSequence` (Esc by default) is used.
+
+````tsx
+
 ## Usage
 
 ```tsx
```

Comment:
Not sure if this markup is expected:
![image](https://github.com/user-attachments/assets/d549842b-3c83-4afd-8181-4e13c6032511)

Reply:
good spot, fixed